examples/js/loaders/GLTFLoader.js

```diff
@@ -1324,16 +1324,16 @@ THREE.GLTFLoader = ( function () {
 		for ( var i = 0; i < dependencies.length; i ++ ) {
 
 			var dependency = dependencies[ i ];
-			var fnName = 'load' + dependency.charAt( 0 ).toUpperCase() + dependency.slice( 1 );
 
 			var cached = this.cache.get( dependency );
 
 			if ( cached !== undefined ) {
 
 				_dependencies[ dependency ] = cached;
 
-			} else if ( this[ fnName ] ) {
+			} else {
 
+				var fnName = 'load' + dependency.charAt( 0 ).toUpperCase() + dependency.slice( 1 );
 				var fn = this[ fnName ]();
```

Comment:
This `else if` check is necessary only for `cameras` but it can be removed (see below).
So we can remove `else if` check and `fnName` can move in `else` statement.